src/index.js

```diff
@@ -475,62 +495,250 @@ const adaptive_test_node = {
 };
 
 // trials to add gamification
-const preload_intro_video = {
+const images = [
+  left_arrow_image,
+  right_arrow_image,
+];
+
+const videos = [
+  intro_video_1,
+  intro_video_2,
+  intro_video_3,
+  game_instructions_1,
+  game_instructions_2,
+  game_instructions_3,
+  pos_instuctions_feedback,
+  neg_instuctions_feedback,
+  two_back_instructions,
+  three_back_instructions,
+  fix_robot_1,
+  fix_robot_2,
+  fix_robot_3,
+  fix_robot_4,
+  fix_robot_5,
+  fix_robot_6,
+  fix_robot_n,
+  generic_game_break,
+];
+
+const preload_videos = {
   type: jsPsychPreload,
-  video: intro_video,
+  video: videos,
 };
 
-const exit_fullscreen = {
-  type: jsPsychFullScreen,
-  fullscreen_mode: false,
-  delay_after: 0,
+const preload_images = {
+  type: jsPsychPreload,
+  image: images,
+};
+
+const video_parameters = {
+  type: videoKeyboardResponse,
+  trial_ends_after_video: false,
+  response_allowed_while_playing: true,
+  trial_duration: null,
+  choices: "ALL_KEYS",
+  width: VIDEO_WIDTH,
+  height: VIDEO_HEIGHT,
+};
+
+const intro_video_node_1 = {
+  stimulus: [intro_video_1],
+  ...video_parameters,
+};
+
+const intro_video_node_2 = {
+  stimulus: [intro_video_2],
+  ...video_parameters,
+};
+
+const intro_video_node_3 = {
+  stimulus: [intro_video_3],
+  ...video_parameters,
+};
+
+// trying to figure out how to make it so that this will push only after the neg_instructions_fb
+// is pushed (and makes sure that they get it right )
+const neg_instuctions_feedback_parameters = {
```

Comment:
To me it seems like this variable is not really being used somewhere. If this is indeed the case, we should remove it.